renovate.json5

```diff
@@ -19,5 +19,13 @@
 
         // Renovate's pre-commit support is still opt-in
         ":enablePreCommit",
-    ]
+    ],
+
+    // update Nix flake.lock file
+    "nix": {
+        "enabled": true
+    },
+    "lockFileMaintenance": {
+        "enabled": true
+    }
```

Comment:
What about `:maintainLockFilesWeekly` preset instead?

https://docs.renovatebot.com/presets-default/#maintainlockfilesweekly

Reply:
Done, thanks for suggestion.